### PR TITLE
Fixes to run on Python3

### PR DESCRIPTION
**Pull request by dstanek**
I've been trying to run all of my tooling on Python 3 even if the projects I work on don't yet support it. This gets that!

### CODE REVIEW

File: next_review.py [hunk 1]
```diff
@@ -13,7 +13,10 @@
 from __future__ import print_function
 
 import argparse
-import ConfigParser
+try:
+    import ConfigParser as configparser
+except ImportError:
+    import configparser
```
Comment (JAORMX) [inline next_review.py]: What about using six?

Comment (dstanek) [inline next_review.py]: I didn't want to add another dependency for such little value. The performance difference of values() vs. itervalues() isn't that compelling for a developer tool. The simple try-except is also easy enough to maintain.

Comment (JAORMX) [inline next_review.py]: Fair enough man, it just looks slightly cleaner. But sure, makes sense.

**Comment by dolph** [comment]
Merged in b0d89492232ba85891621ce19c97c01766191397

**Pull request closed without merging**